### PR TITLE
Changed a tiles  width in marketplace and prices updated

### PR DESCRIPTION
**Pull request by areksolek**
Changed a tiles  width in marketplace and prices updated

### CODE REVIEW

**Comment by mariuszkrzaczkowski** [comment]
## Migration Recommended

It appears you are still using the CoPilot OAuth App - we recommend you [migrate to the GitHub App](https://github.com/apps/blackduck-copilot/installations/new/permissions?suggested_target_id=10201412&repository_ids[]=28060246)

**Comment by mariuszkrzaczkowski** [comment]
## Black Duck Security Report

Merging #11321 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/11321)

File: public_html/layouts/basic/skins/gray/layout/_Base.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
 /* {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} */
 
 .mainBody {
-	background-color: #d1d1db;
+	background-color: $yf-gray-bg;
+}
+.Vtiger_Index {
```
Comment (rskrzypczak) [inline public_html/layouts/basic/skins/gray/layout/_Base.scss]: ?

File: public_html/layouts/basic/styles/layout/_Base.scss [hunk 2]
```diff
@@ -7,6 +7,17 @@
 	width: 100%;
 	overflow-x: hidden;
 }
+.Vtiger_Index {
```
Comment (rskrzypczak) [inline public_html/layouts/basic/styles/layout/_Base.scss]: ?

**Pull request merged**